app/featureswitch/core/config/FeatureSwitchingModule.scala

```diff
@@ -49,3 +49,7 @@ case object ReturnToLegacy extends FeatureSwitch {
   override val displayName: String = "Return the User to the Legacy EMCS service"
 }
 
+case object ShortageExcessFlow extends FeatureSwitch {
```

Comment:
This is really minor, but do you think we should call this `NewShortageExcessFlow`?
```suggestion
case object NewShortageExcessFlow extends FeatureSwitch {
```

Reply:
updated as per review comment